### ISSUE TITLE
JIT functional problems

### ISSUE THREAD

**Issue by DDCHINA0** [leminlimez/Cowabunga#292]
iPhone13Pro
System version: iOS15.2
When running APP JIT Enabler, the corresponding software has been opened in the background, and it will still flash back during the opening process.

**Comment by AlbionBisha123** [comment]
hello JIT enabler doesent work iphone se 2020 16.1.2 did everything correctly with app in backround but when it tries to enable JIt cowabunga crashes it dosent work pls reply

**Comment by leminlimez** [comment]
the disk needs to be mounted using a computer before it will work

**Comment by ozonostudio** [comment]
JIT crash on iPhone 12 Pro Max when it’s trying to enable JIT on any app

**Comment by DDCHINA0** [comment]
> 磁盘需要使用计算机安装才能工作

Is there a relevant tutorial, or does this function need to be connected to a Mac?

**Comment by leminlimez** [comment]
it should work with any operating system using ideviceimagemount

**Comment by ozonostudio** [comment]
> it should work with any operating system using ideviceimagemount

Yeah I don't even know what are you talking about, not everyone are that into tech stuff, some of us just search for tutorials or manuals

**Comment by mjd180** [comment]
> > it should work with any operating system using ideviceimagemount
> 
> Yeah I don't even know what are you talking about, not everyone are that into tech stuff, some of us just search for tutorials or manuals

What kind of reply is this to the developer? Ignorance is no excuse for poor manners.

**Comment by ozonostudio** [comment]
> > > it should work with any operating system using ideviceimagemount
> > 
> > 
> > Yeah I don't even know what are you talking about, not everyone are that into tech stuff, some of us just search for tutorials or manuals
> 
> What kind of reply is this to the developer? Ignorance is no excuse for poor manners.

Poor manners? how are you reading this? you know this are just words right I'm not adding anything rude or disrespectful.
"I don't know what are you talking about" - 100% I don't know what or how ideviceimagemount works on cowabunga, because everything I find don't have anything to do with the app itself.
"not everyone are that into tech stuff" - That's true not everyone know how to develop anything are just final users that want to see a tutorial or some guide just to understand anything.
"some of us just search for tutorials or manuals" - Again, some of us need manuals or video guides

Don't know if you read in you mind this with a "fuck you developer" or something like that, but man you need to understand if somebody it's actually trying to be mean before say "poor manners" also depending on the country of each person some people just talk straight and even consider the "don't worry" like a rude way to reply. remember you're on internet, not everyone talk or reply on the same way and that doesn't mean they're being rude.

**Comment by leminlimez** [comment]
> > it should work with any operating system using ideviceimagemount
> 
> Yeah I don't even know what are you talking about, not everyone are that into tech stuff, some of us just search for tutorials or manuals

read this github: https://github.com/haxi0/DirtyJIT/releases

he explains how to use it but also the how to in the app should detail it too

**Comment by CycloKid** [comment]
never seen the instructions. when i try to select the “show instructions again” button, nothing happens.

**Comment by leminlimez** [comment]
nathan brought this to my attention, i will look into fixing it when i have the time
